### PR TITLE
Add Statement to Clarify Preferred Image Policy

### LINKED ISSUE #1204: Image Support Policy and Golden Images
**Affected Document:**

Runtime/AC Policy Support, like this: https://docs.astronomer.io/software/ac-support-policy#astronomer-certified-maintenance-policy

**Problem Type (Priority):**

- [ ] Error (Critical): This document has existing information that's incorrect and prevents readers from completing the documented setup (Add the `error/critical` label to this issue)
- [x] Misleading (Important): This document could benefit from additional context, guidance, or clarity (Add the `error/FYI` label to this issue)
- [ ] Typo (Low): Any issue related to spelling/ punctuation/ formatting (Add the `error/typo` label to this issue)

**Suggested Change (include links to relevant existing docs, support tickets, messages, or external content):**

Hoping to add something like this after the content about upgrades being the first line of defence. 

```
Some customers choose to restrict their deployments to only use organization-approved images. While we understand the reasoning for this, because our primary means to reacting to bugs is to put out patch releases as new images, this policy necessarily restricts Astronomer's ability to support you. Since patch releases are the only viable way to respond to bugs, if you encounter a bug, our support team may direct you to upgrade your Airflow version, including going through any organization-approval process your organization may impose.
```

**Additional Notes:**

Large customers seem to have an expectation that we can somehow fix things without requiring them to upgrade Airflow. This mostly affects orgs where data teams use "blessed" golden images approved by other IT teams. This creates some friction. While we have no control over their policies, this at least gives CRE a place to point to that explains consequences. 

Reported by @collinmcnulty

### PR DESCRIPTION
**Pull request by kwcanuck**
Resolves #1204 

@cmarteepants I have created a preliminary draft of the text you requested. When time permits, please review and provide your edits or approval. Thanks!

### CODE REVIEW

**Review by cmarteepants — APPROVED** [review]
Love it. Thanks!

File: software/ac-support-policy.md [hunk 1]
```diff
@@ -103,6 +103,8 @@ The maintenance period for an Astronomer Certified version depends on its releas
 
 For each `major.minor` pair, only the latest patch and hot-fix combination is supported at any given time. If you report an issue with an Astronomer Certified patch or hot-fix version that is not latest, the Astronomer Support team will always ask that you upgrade as a first step to resolution. For example, the Support team encourages any user who reports an issue with Astronomer Certified 2.2.2 to first upgrade to 2.2.3 as soon as it's generally available.
 
+To correct bugs, Astronomer provides patch releases as new images and recommends that organizations install them regularly to ensure optimum performance. Failure to install patch releases as they are made available can make it challenging for Astronomer support to assist organizations that restrict their deployments to organization-approved images. To resolve issues, organizations are often required to upgrade their Airflow version. The upgrade and image approval process could significantly delay the resolution of issues.
```
Comment (jwitz) [inline software/ac-support-policy.md:106]: I'm a little confused about why we need this paragraph when the line before communicates something very similar: 

>  If you report an issue with an Astronomer Certified patch or hot-fix version that is not latest, the Astronomer Support team will always ask that you upgrade as a first step to resolution. For example, the Support team encourages any user who reports an issue with Astronomer Certified 2.2.2 to first upgrade to 2.2.3 as soon as it's generally available.

Comment (kwcanuck) [reply software/ac-support-policy.md:106]: @jwitz my understanding is that @cmarteepants wanted to go deeper and speak directly to organizations that limit their deployments to images that they have approved. Apparently, this is an issue that needed to be addressed. I will ask Constance to confirm.

Comment (cmarteepants) [reply software/ac-support-policy.md:106]: The request was from @collinmcnulty. Agreed that it's already said, but he wanted us to be more explicit so CRE could point to this when customers complain that we are telling them to upgrade.

File: software/ac-support-policy.md [hunk 2]
```diff
@@ -103,6 +103,8 @@ The maintenance period for an Astronomer Certified version depends on its releas
 
 For each `major.minor` pair, only the latest patch and hot-fix combination is supported at any given time. If you report an issue with an Astronomer Certified patch or hot-fix version that is not latest, the Astronomer Support team will always ask that you upgrade as a first step to resolution. For example, the Support team encourages any user who reports an issue with Astronomer Certified 2.2.2 to first upgrade to 2.2.3 as soon as it's generally available.
```
Comment (jwitz) [inline software/ac-support-policy.md:104]:
```suggestion
To fix bugs, Astronomer provides patch releases as new images and recommends that organizations install them regularly to ensure optimum performance. Upgrading only when an issue arises can result in delays to resolving the issue. 

For each `major.minor` pair, only the latest patch and hot-fix combination is supported at any given time. If you report an issue with an Astronomer Certified patch or hot-fix version that is not latest, the Astronomer Support team will always ask that you upgrade as a first step to resolution. For example, if Astronomer Certified 2.2.3 is the latest patch in the 2.2.x series, the Support team would encourage any user who reports an issue with Astronomer Certified 2.2.2 to first upgrade to 2.2.3 before starting to troubleshoot the issue. 
```

File: software/ac-support-policy.md (hunk 1 above)
Comment (jwitz) [inline software/ac-support-policy.md:106]:
```suggestion
```

**Review by jwitz — COMMENTED** [review]
I tried rewording this a bit so that we combined the more stern warning that Collin recommended with the original paragraph. I also tried to soften the language just a bit so that we aren't admonishing users or making them feel a burden for not installing the latest patch.

**Comment by kwcanuck** [comment]
@jwitz can you review and unblock this PR? Thanks.

**Review by jwitz — APPROVED** [review]

**Pull request merged**